agenda/presentation/src/main/java/com/pronoidsoftware/agenda/presentation/detail/components/event/photo/previews/EventDetailPhotoHolderPreviews.kt

```diff
@@ -0,0 +1,117 @@
+package com.pronoidsoftware.agenda.presentation.detail.components.event.photo.previews
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.pronoidsoftware.agenda.presentation.R
+import com.pronoidsoftware.core.presentation.designsystem.PlusIcon
+import com.pronoidsoftware.core.presentation.designsystem.TaskyLightBlue2
+import com.pronoidsoftware.core.presentation.designsystem.TaskyTheme
+
+// For previews and screenshot testing
+// previews don't handle URIs
+@Composable
+private fun PhotoBorderFromResId(
```

Comment:
The border should be pretty independent from a Uri, right? You're at least not using any res ID or Uri in this composable